### ISSUE TITLE
Associate's Corpse decays to Trace with associate aspect

### ISSUE THREAD

**Issue by user18130814200115-2** [weatherfactory/cultistsimulator-visible#78]
On JENUFA (not tested on other versions) Windows 10, Exile Legacy:
If one of your associates is killed by the foe,
they become a corpse with the "associate" aspect.
This corpse will then decay to a Trace with the same aspect.

The trace card can then be used to defend against the foe's attacks resulting in a perfect guard with no cool down.
Also if your "associate" (trace) is hurt, it will wound the foe, and give you the trace back.
In other words, this broken trace guarantees a victory.
